scripts/build.sh

```diff
@@ -25,9 +25,9 @@ echo "Previous client cleaned."
 
 echo "Build entry point..."
 mkdir -p ./build/server/views
-jade --client --no-debug --hierarchy --out ./build/server ./server
-echo "var jade = require('jade/runtime');module.exports=" | \
-    cat - ./build/server/views/index.js > ./build/server/views/index.js.tmp
+pug --client --no-debug --out ./build/server ./server
```

Comment:
> I'm tired 😫

😄 

The priority comes to the first available in the `$PATH`ENV variable, so with to top `export` instruction, your path become something like `./node_modules/.bin:/usr/bin:/bin`. So command is first seeking into the local `node_modules` binary dir, then fallback to `/usr/bin`(which contains generally links to globally installed node modules) and so on. No more headaches 😉